jib-testing-common/build.gradle

```diff
@@ -0,0 +1,16 @@
+plugins {
+  id 'java'
+}
+
+// disable javadoc on this project, it's only for testing
+tasks.javadoc.enabled = false
+
+dependencies {
+  // if you want to use classes that have dependencies in the main source
+  // sets of other projects, mark them compile only
+  compileOnly project(':jib-plugins-common')
+  compileOnly project(':jib-core')
```

Comment:
I don't know how this project will be used, but will source files in this project need using classes from `jib-plugins-common` and `jib-core`?

What's weird to me is that `jib-core` depends on `jib-testing-common` (albeit via `testImplementation`) but `jib-testing-common` depends on `jib-core`.

Reply:
`jib-core:test` can be seen as it's own project with a dependency on `jib-core`. Since the common testing infrastructure is bound to reference production classes, I think this is necessary. But if the question is: Is this PR necessary? I think it's convenient.